1-JavaScript/Week4/homework/ex5-wallet/index.js

```diff
@@ -77,7 +77,7 @@ const quiz = {
       b: 'cash, name', 
       c: 'amount, this, wallet'
     },
-    answer: undefined,
+    answer: c,
```

Comment:
@Mehran-Banbersta When you run the test for this exercise it fail because the answer should be a string:
```suggestion
    answer: 'c',
```
Also think a little bit more about it the answer is not c